src/github.com/getlantern/fronted/test_support.go

```diff
@@ -0,0 +1,30 @@
+package fronted
+
+import (
+	"crypto/x509"
+	"testing"
+
+	"github.com/getlantern/keyman"
+)
+
+// ConfigureForTest configures fronted for testing using default masquerades and
+// certificate authorities.
+func ConfigureForTest(t *testing.T) {
```

Comment:
We had a bunch of tests that all duplicated masquerade configuration.  That was annoying and the tests were starting to fail because they had out of date masquerades.  I've moved the default masquerades into the fronted package and provide a convenience ConfigureForTest method that any tests relying on fronted functionality can just use.